clients/ops/admin-ui/src/app/store.ts

```diff
@@ -38,14 +73,33 @@ const reducer = {
   connectionType: connectionTypeReducer,
 };
 
+const rootReducer = (state: any, action: AnyAction) => {
+  if (action.type === "auth/logout") {
+    storage.removeItem(STORAGE_ROOT_KEY);
+    // eslint-disable-next-line no-param-reassign
+    state = {};
```

Comment:
Rather than mutating directly like this, could we do something like `const newState = {}` and then pass that to the `combineReducers` below?